src/vma/dev/ring_tap.cpp

```diff
@@ -179,7 +179,7 @@ void ring_tap::tap_create(net_device_val* p_ndev)
 
 	/* Disable Ipv6 for TAP interface */
 	snprintf(command_str, TAP_STR_LENGTH, TAP_DISABLE_IPV6, tap_name);
-	if (run_and_retreive_system_command(command_str, return_str, TAP_STR_LENGTH)  < 0) {
+	if (run_and_retreive_system_command(command_str, return_str, TAP_STR_LENGTH)  < 0 && errno != ECHILD) {
```

Comment:
I think that this suppression should be a part of `run_and_retreive_system_command` as
```
		// Check exit status code
		rc = pclose(file);
                if (rc == -1 && errno ==ECHILD) {
                 /* suppress a case when termination status can be unavailable to pclose() */
rc = 0;
                }
```
See https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.bpxbd00/rpclos.htm